src/lib/isObjEmpty.js

```diff
@@ -0,0 +1,5 @@
+function isObjEmpty(obj) {
+  return Object.keys(obj).length === 0;
+}
+
+export default isObjEmpty;
```

Comment:
Empty objects are falsey when evaluated by a conditional. This function might not be necessary.

Reply:
I don't believe this to be the case. When experimenting with the following code: `if ({}) console.log('truthy');`, the console logs "truthy". As I understand it, the only six falsey values in JavaScript are `undefined`, `null`, `NaN`, `0`, an empty string and `false`.

Reply:
Ah, I stand corrected! I thought I've used it in this way before, but I guess not 🤯 Good catch. Thanks for the knowledge nugget @mjhughes707 !

Reply:
I had to research that one, 'cause on first glance, I totally just figured you were right and I had overcomplicated it. 😂